frontend/src/components/NavBar/index.tsx

```diff
@@ -46,11 +51,31 @@ import PanelButton from './PanelButton';
 
 const { alertFormActive, header } = appConfig;
 
+const getAvailableDashboards = (): PanelItem[] => {
+  const configuredReports = getConfiguredReports();
+  return configuredReports.map((report, index) => ({
+    panel: Panel.Dashboard,
+    label: report.title,
+    icon: <SpeedOutlined />,
```

Comment:
Looks like the icon in the Figma mockups isn’t available in this version of Material icons